libevm/VM.cpp

```diff
@@ -815,7 +815,93 @@ void VM::interpretCases()
 			m_sp -= 2;
 		CASE_END
 
+		CASE_BEGIN(JUMPTO)
+		THROW_IF_SUBS_DISABLED();
+			ON_OP();
+			updateIOGas();
+			{
+				uint64_t dest       = m_code[m_pc++];
+				dest =  (dest << 8) | m_code[m_pc++];
+				dest =  (dest << 8) | m_code[m_pc++];
+				dest =  (dest << 8) | m_code[m_pc++];
+				m_pc =    dest;
+			}
+		CASE_END
+
+		CASE_BEGIN(JUMPIF)
+		THROW_IF_SUBS_DISABLED();
+			ON_OP();
+			updateIOGas();
+			if (*m_sp--) {
+				uint64_t dest       = m_code[m_pc++];
```

Comment:
Good idea, unless we change to a different encoding.  @Arachnid didn't like this one.  But a helper is still a good idea.